docs/guides/postinstall.md

```diff
@@ -172,9 +174,40 @@ S3 suspend has been broken since macOS Sonoma, it has never been fixed, but this
      [Install]
      WantedBy=sleep.target
      ```
+3. Check your modprobe location
+     ```bash
+     which modprobe
+     which rmmod
+     ```
+and fix the pathes in service script if they differ for your system. If you having problems with touchbar being dead after restoring state from suspend then uncomment ExecStartPre line.
 
-3. Enable the service by running: `sudo systemctl enable --now suspend-fix-t2.service`
+4. Enable the service by running: `sudo systemctl enable --now suspend-fix-t2.service`
 
 !!! note
-    This seems to be working only on Arch with `CONFIG_MODULE_FORCE_UNLOAD=y` in the kernel config.
-    To check, run: `zcat /proc/config.gz | grep "CONFIG_MODULE_FORCE_UNLOAD"`
+
+    This seems to be working with `CONFIG_MODULE_FORCE_UNLOAD=y` in the kernel config.    
+    To check, run: `zcat /proc/config.gz | grep "CONFIG_MODULE_FORCE_UNLOAD"` on arch or `cat /boot/config-$(uname -r) | grep "CONFIG_MODULE_FORCE_UNLOAD"` on Debian-based distros.
+
+Without this config option you wont be able to unload the required modules, they will be busy.
+
+5. If touchbar occasionally does not work on boot but works after suspend+restore then you can place this workaround somewhere late after boot
+     ```bash
+     modprobe -r hid_appletb_kbd
+     modprobe -r brcmfmac_wcc
+     modprobe -r brcmfmac
+     rmmod -f apple-bce
+     
+     sleep 1
+     
+     modprobe apple-bce
+     modprobe brcmfmac
+     modprobe brcmfmac_wcc
+     
+     touchbar --restart
```

Comment:
touchbar --restart is an Ubuntu only command. Make it disable and enable tiny-dfr.service instead.